pkg/skaffold/kubernetes/log.go

```diff
@@ -116,7 +116,9 @@ func (a *LogAggregator) Start(ctx context.Context) error {
 
 // Stop stops the logger.
 func (a *LogAggregator) Stop() {
-	a.cancel()
+	if a.cancel != nil {
```

Comment:
now that we're touching this: `a.cancel` can be `nil` when it's not yet started, right? 
Also, what happens when we call Stop() twice? I.e. the cancelFunc called twice? 
What if instead of `nil` the "default" value of the cancel is a "noop" func, i.e `cancel: func() {}` in NewLogAggregator - and then still, `Stop`will have to remember to set it back to `func() {}` - this way we don't have to check for `nil`.
Alternatively we can keep it as it is and have `a.cancel=nil` represent the Stopped state -  but then we have to set `a.cancel = nil` here. WDYT?

Reply:
Let me handle that in another PR. Current state is that `cancel` can be called multiple time, which is a noop passed the first time.